docs/articles/send-money/The-Free-Plan.md

```diff
@@ -21,15 +21,7 @@ The free plan is ideal for start-ups and small businesses to manage expenses. Wi
 - Navigate to new.expensify.com, enter your company email address, and set a password
 - Click the **green “+”** button and select **_New workspace_**
 
-<img width="400" src="/assets/images/new-workspace.png">
-
-Once you’ve created your Workspace, you should receive a message from Concierge encouraging you to chat with your Setup Specialist.
-
-<img width="800" src="/assets/images/chat-with-concierge-setup.png">
```

Comment:
Ah I think we should also remove these images from the `/assets/images` folder

Reply:
Ah, good call. I'll do that now.